### PR TITLE
api: copy '/health' and '/diagnose' to '/api/{version}' and deprecate old ones

### PR DESCRIPTION
**Pull request by Luffbee**
### What problem does this PR solve? 

Fix api document: `/health` and `/diagnose` don't  need `/api/{version}` prefix.
However the new api.html is a little ugly, but I didn't find an easy way to make it beautiful.

### Check List 

Tests 
 - Manual test

### CODE REVIEW

**Comment by nolouch** [comment]
Cool! If you want to do more beautiful, you can pick a theme in this [list](https://www.npmjs.com/search?q=keywords:raml2html-theme).

**Comment by nolouch** [comment]
And there is a [image](https://hub.docker.com/r/bbrodriges/raml2html/dockerfile ) that supports custom theme.

**Comment by disksing** [comment]
em, how about moving them to /api too?

**Comment by Luffbee** [comment]
> em, how about moving them to /api too?
I thought that's a better solution, but maybe somebody is using them.
Perhaps we can "copy" them to /api? And hide the old ones in document?

**Comment by disksing** [comment]
@Luffbee Good idea. We can remove obsolete APIs some day in the future.

File: server/api/api.raml [hunk 1]
```diff
@@ -1,1197 +1,1199 @@
...
-          description: The log level is updated.
-        400:
-          description: The input is invalid.
-        500:
-          description: PD server failed to proceed the request.
-
-
-/classifier:
-  description: The namespace classifier. Methods depend on current classifier.
+#%RAML 1.0
```
Comment (shafreeck) [inline server/api/api.raml]: Why are all lines changed?

Comment (Luffbee) [reply server/api/api.raml]: I added one more level to all apis under /api/{version}, so almost all lines added one more indent.
However, this won't be a problem any more. I'm going fix the api by the "copy" idea mentioned above, which don't need to modify the docs.

**Comment by Luffbee** [comment]
> LGTM, but we need to update the title.

title updated

**Pull request merged**